### ISSUE TITLE
Plan to release IsaacGym version's hanover-sim environment

### ISSUE THREAD

**Issue by HanzhiC** [NVlabs/handover-sim#15]
Hi,

Thanks for building such a nice simulator for handover research.
I read from the paper that you have the IsaacGym version, but when I followed the instructions from Easysim to modify parameters in config files and tried to use IsaacGym, the physical properties of hands, robots and objects seemed to be very wrong.  So could you provide an official config file for the IsaacGym environment?

**Comment by ychao-nvidia** [comment]
Thanks. The IG version is not ready for release yet and we currently don't have a timeline for that.

What kind of use case are you thinking for the IG version? E.g. just for evaluation, or also training potentially? FWIW, in our latest [CVPR'23 paper](https://handover-sim2real.github.io/) we use it for evaluation only.

**Comment by HanzhiC** [comment]
Thanks for the reply. I would like to use IG for both training and evaluation. 
Yes I noticed that you evaluate method in CVPR 23 in IG. 
May I ask if there is still any plan for code release of the CVPR'23 paper, potentially with the evaluation environment you built in IG?

**Comment by ychao-nvidia** [comment]
Yes, our CVPR'23 code will be out very soon. This release will include everything we have for the Bullet env. We are also keen in releasing the IG env but no timeline on that yet.

**Comment by ychao-nvidia** [comment]
Our CVPR'23 code is live now [here](https://github.com/NVlabs/handover-sim2real).
